### PR TITLE
Sig Governance Short Template

### PR DESCRIPTION
**Pull request by pwittrock**
Short template for sig governance

Also see:

- Long template #1650 
- ~Requirements PR: #1800~ Merged

### CODE REVIEW

**Comment by philips** [comment]
LGTM, but I see you are still incorporating stuff.

**Comment by pwittrock** [comment]
@philips @timothysc Thanks!

I updated the PR to include the new SIG Lead name (long live Coordinator!).  I also refined / clarified some points that were implied but could be made more clear.

- Change SIG Lead -> Coordinator
- Generally tighten up wording
- Clarify that votes to select coordinators and project leads are done amongst existing coordinators / project leads
- Re-order options for SIG Technical Lead vs Federation of Project Leads to put SIG TL first since it is expected to be the more common case
- Clarify that subproject leads as escalation point a *MUST* (this wasn't clearly called out)
- Clarify that subproject leads can delegate decision making authority
- Clarify that coordinators can delegate responsibilities to members
- Clarify SIG membership as having some documented role in either the SIG or a subproject (reviewer / approver count)

File: committee-steering/governance/sig-governance-template-short.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    and followed up during the next SIG meeting.
+
+Issues impacting multiple subprojects in the SIG should be resolved by either:
+
+- Option 1: SIG Technical Leads
+- Option 2: Federation of Subproject Leads
+
+[lazy-consensus]: http://communitymgt.wikia.com/wiki/Lazy_consensus
+[super-majority]: https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote
+[warnocks-dilemma]: http://communitymgt.wikia.com/wiki/Warnock%27s_Dilemma
```
Comment (tpepper) [inline committee-steering/governance/sig-governance-template-short.md]: These links look copied from the long template, with some not used and probably should be (eg: lazy consensus) and some not used and probably need removed (eg: warnocks dilemma and slo).

Comment (pwittrock) [reply committee-steering/governance/sig-governance-template-short.md]: Done

File: committee-steering/governance/sig-governance-template-short.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  - Number: 2-3
+  - Defined in [sigs.yaml]
+
+
+- *Optional Role*: SIG Technical Leads
+  - Established new and shutdown existing subprojects
+  - Resolve X-Subproject technical issues and decisions
+
+
+- Subproject Leads
```
[the remark this answers is not in the document]
Comment (pwittrock) [reply committee-steering/governance/sig-governance-template-short.md]: +1  Done

**Comment by pwittrock** [comment]
Addressed outstanding comments.  Updated the SIG role names.

**Pull request merged**